Modules/Devices/src/Devices.Application/Extensions/IServiceCollectionExtensions.cs

```diff
@@ -18,7 +19,7 @@ public static void AddApplication(this IServiceCollection services)
         services.TryAddTransient(typeof(IPipelineBehavior<,>), typeof(LoggingBehavior<,>));
         services.TryAddTransient(typeof(IPipelineBehavior<,>), typeof(RequestValidationBehavior<,>));
         services.AddAutoMapper(typeof(AutoMapperProfile).Assembly);
-        services.AddValidatorsFromAssembly(typeof(RegisterDeviceCommandValidator).Assembly);
+        services.AddValidatorsFromAssembly(typeof(DeleteClientCommandValidator).Assembly);
```

Comment:
I mentioned this is our call, I wasn't sure if the issue with the validators not running was the one we had talked about previously (with how the pipelines are registered) or if there was something wrong with my specific validator.
I'll leave it like this anyways since it doesn't matter, just wanted to clarify 👍🏼

Reply:
Okay, got it. :)